search-app/src/cmr/search/site/pages.clj

```diff
@@ -0,0 +1,36 @@
+(ns cmr.search.site.pages
+  "The functions of this namespace are specifically responsible for returning
+  ready-to-serve pages."
+  (:require
+    [cmr.search.site.data :as data]
+    [selmer.parser :as selmer]))
+
+(defn render-template-ok
+  "A utility function for preparing template pages."
+  [template data]
+  {:status 200
+   :body (selmer/render-file template data)})
+
+(defn home
+  "Prepar the home page template."
+  [request]
+  (render-template-ok
+    "templates/index.html"
+    (data/get-index request)))
+
+(defn landing-links
+  "Prepare the page that links to all landing page links.
+
+  For now, this is just a page with a single link (the EOSDIS collections
+  landing pages)."
+  [request]
+  (render-template-ok
+    "templates/landing-links.html"
+    (data/get-landing-links request)))
+
+(defn eosdis-landing-links
+  "Prepare the page that provides links to all the EOSDIS landing pages."
+  [request]
+  (render-template-ok
+    "templates/eosdis-landing-links.html"
```

Comment:
There's a really interesting discussion of this in a Clojure Style Guide ticket:
 * https://github.com/bbatsov/clojure-style-guide/issues/126

It turns out this rule is actually carried over from Lisp, though it has been very inconsistently used in the Lisp community (no surprises there). One commenter gives some exceptions: "SICP, Graham's ANSI Common Lisp, Graham's On Lisp, Norvig's PAIP, and Hoyt's Let Over Lambda, Colin Jones's Mastering Clojure Macros, Fogus and Chouser's Joy of Clojure" -- the fact that SICP and PAIP are two of my favourite and well-worn Lisp books explains why I have adopted the 2-space "convention".

(Btw, this is not an argument for moving to 2-space indents on args :-) Just sharing some fun discovery ...)